modules/hello-world/pages/overview.adoc

```diff
@@ -17,34 +17,25 @@ The Couchbase Python SDK allows Python applications to access a Couchbase cluste
 The Python SDK uses the high-performance C library libcouchbase to handle communicating to the cluster over Couchbase's binary protocols.
 
 
-////
 [.column]
 [.content]
 [source,python]
 ----
-from twisted.internet import reactor
-from txcouchbase.cluster import TxCluster
+from couchbase.cluster import Cluster
+from couchbase.auth import PasswordAuthenticator
 
-cluster = TxCluster('couchbase://localhost', ClusterOptions(PasswordAuthenticator('username', 'password')))
```

Comment:
Ah, this is because the Twisted ("TxCluster") support is an optional thing but not the first, most important detail to call out? Is it still documented elsewhere?

Reply:
Exactly!  I think overtime we move to highlight async (specifically asyncio), but I still need some time to clean up the async 'things'.  At the moment I do not think we should highlight them until they are properly vetted.